src/algorithms/PVT/gnuradio_blocks/rtklib_pvt_gs.cc

```diff
@@ -511,11 +518,14 @@ rtklib_pvt_gs::rtklib_pvt_gs(uint32_t nchannels,
     d_beidou_dnav_almanac_sptr_type_hash_code = typeid(std::shared_ptr<Beidou_Dnav_Almanac>).hash_code();
 
     d_start = std::chrono::system_clock::now();
+
+    d_spoofing_detector = spoofing_detector;
```

Comment:
if you initialize this in the member initializer list, you at least have the opportunity to elide one of the copies.